### PR TITLE
fix: AIS2 Cosign login 

### PR DESCRIPTION
**Pull request by mrshu**
* Up until now there was a difference between how AIS2's login has been
  set up on BETA vs. production. This difference has ceased to exist,
  with new endpoint for login via Cosign being created.

* This commit reflects the situation by updating VOTR to the newest
  version which is capable of handling the new login setup and
  refactoring how the login is handled when the application form is
  being submitted to AIS in both BETA and production.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

File: eprihlaska/views.py [hunk 1]
```diff
@@ -1007,7 +1007,7 @@ def admin_ais_test():
     ctx = create_context(cosign_cookies,
                          origin='ais2.uniba.sk')
     # Do log in
-    ctx.request_html('/ais/login.do', method='POST')
+    ctx.request_html('/ais/loginCosign.do', method='POST')
```
Comment (Adman) [inline eprihlaska/views.py:1010]: I would say this could go to settings.

Comment (mrshu) [reply eprihlaska/views.py:1010]: That's indeed true but at the end of the day, these values are basically hardcoded in AIS, so I think we can live with these values being hardcoded as well :)

Comment (Adman) [reply eprihlaska/views.py:1010]: Not sure what "hardcoded in AIS" means, but we are using it on 2 places here.

**Comment by tvinar** [comment]
Mne sa tiez zda, ze by to mohlo pekne poputovat do configu.

File: eprihlaska/views.py (hunk 1 above)
Comment (mrshu) [reply eprihlaska/views.py:1010]: What I meant was that `loginCosign.do` is extremely unlikely to change and so extracting it to config does not make too much sense to me (just like we did not extract `login.do` to its own config variable).

But if you think it's worth it, I am happy to refactor this and put it to config :slightly_smiling_face:

**Pull request merged**